src/xcode/ENA/ENA/Source/Views/BaseElements/ENAButton.swift

```diff
@@ -108,6 +122,18 @@ class ENAButton: DynamicTypeButton {
 	private func applyHighlight() {
 		highlightView.isHidden = !isHighlighted
 	}
+
+	func startSpinner() {
```

Comment:
Please expose `isLoading` property to like `isTransparent` and `isInverted` for use in InterfaceBuilder.